app/gui/language/ast/impl/src/lib.rs

```diff
@@ -1194,6 +1170,18 @@ impl<T: HasTokens> HasRepr for T {
         self.feed_to(&mut consumer);
         consumer.repr
     }
+
+    fn len(&self) -> Bytes {
+        let mut consumer = LengthBuilder::default();
+        self.feed_to(&mut consumer);
+        consumer.length
+    }
+
+    fn char_count(&self) -> Codepoints {
```

Comment:
Would it be more intuitive to call it `codepoin_count`?

Reply:
I thought `char` is better recognizable by rust developers. However, now I see the char is not exactly the code point: I will investigate this.

However, as I see, it is rather `Codepoints` a name we would change to `Chars`